### DISCUSSION TITLE
Invitation not found

### DISCUSSION THREAD

**Discussion by Sidradeveloper** [community/community#51695] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi .. I am not getting invite which my team has sent to me. I have checked my email and notification . Rather i am getting 404 error

please help me out.

**Comment by osalbahr** [comment]
Same here

**Comment by TheRealSamChaney** [comment]
Yes I got the invitation email and when I clicked the green "Join :org name:" button, it just brought me to a page that said "Invitation not fount". It was just sent like 5 minutes earlier so it was not expired, and I am certain that I was already logged in to the correct GitHub account at the time when I clicked the button.
